source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/Mappers/WholesaleResult/ChargeTypeMapper.cs

```diff
@@ -30,4 +30,16 @@ chargeType switch
                 actualValue: chargeType,
                 "Value does not contain a valid string representation of a charge type."),
         };
+
+    public static string ToDeltaTableValue(ChargeType chargeType) =>
+        chargeType switch
+        {
+            ChargeType.Tariff => "tariff",
+            ChargeType.Fee => "fee",
+            ChargeType.Subscription => "subscription",
+            _ => throw new ArgumentOutOfRangeException(
+                nameof(chargeType),
+                actualValue: chargeType,
+                "Cannot map ChargeType to delta table value"),
```

Comment:
```suggestion
                $"Cannot map typeof{ChargeType} to delta table value"),
```
Would this make it easier to maintain, if the type changes name?

Same for: "ResolutionMapper.cs"